support/macos/nix-builder/entrypoint.py

```
@@ -0,0 +1,68 @@
+			'ssh-keygen',
+			'-t',
+			'ed25519',
+			'-N',
+			'',
+			'-f',
+			str(vol_server_key),
+			'-C',
+			'nix-builder',
+		]
+	)
+	os.chmod(vol_server_key, 0o666)
```

Ruff (0.15.12)

[error] `os.chmod` setting a permissive mask `0o666` on file or directory

(S103)